src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/controller/v1_0/DisputeController.java

```diff
@@ -423,4 +423,27 @@ public ResponseEntity<DisputantUpdateRequest> updateDisputantUpdateRequestStatus
 		return new ResponseEntity<DisputantUpdateRequest>(disputantUpdateRequest, HttpStatus.OK);
 	}
 
+	/**
+	 * PUT endpoint that updates the status of one or more DisputantUpdateRequest records to PENDING from HOLD.
+	 *
+	 * @param List of IDs of DisputantUpdateRequest records' status to be updated to pending
+	 */
+	@PutMapping("/dispute/updateRequest/{ids}/pending")
```

Comment:
Not sure I follow this. Isn't "ids" here a list of ids? It may be better to move this to a parameter instead of a path variable.

Reply:
Yes, the "ids" here is a list of ids of the DisputantUpdateRequest that needs to be updated. I think getting the ids from the path variable makes more sense since we are not passing a parameter to filter or sort items but rather identifying the resources to update. Also, it is more in line with the convention of querying other items in other endpoints using single id which we are passing as path variable.